snapshot/snapshot.go

```diff
@@ -631,9 +649,8 @@ func (o *snapshotter) workPath(id string) string {
 // Close closes the snapshotter
 func (o *snapshotter) Close() error {
 	// unmount all mounts including Committed
-	const cleanupCommitted = true
 	ctx := context.Background()
-	if err := o.cleanup(ctx, cleanupCommitted); err != nil {
+	if err := o.cleanup(ctx, o.cleanupCommitted); err != nil {
```

Comment:
We should not change the default behaviour here. cleanupCommitted should be enabled by default and disabled by an option.